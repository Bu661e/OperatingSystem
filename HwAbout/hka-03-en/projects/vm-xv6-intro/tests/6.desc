test mprotect in child process
