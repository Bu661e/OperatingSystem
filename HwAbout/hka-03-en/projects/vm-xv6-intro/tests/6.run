cd src; ../../tester/run-xv6-command.exp CPUS=1 Makefile.test test_6 |  grep -e XV6_TEST_OUTPUT -e trap; cd ..
