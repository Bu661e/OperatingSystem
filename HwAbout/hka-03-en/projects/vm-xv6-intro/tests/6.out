XV6_TEST_OUTPUT mprotect returned 0
pid 4 test_6: trap 14 err 7 on cpu 0 eip 0x1048 addr 0x1000--kill proc
